src/autostep/Program.cs

```diff
@@ -46,5 +47,13 @@ static async Task<int> Main(string[] args)
 
             return await result.InvokeAsync();
         }
+
+        private static bool DisplayHelp(string[] args) =>
+            NoArgsPassed(args) ||
+            NewCmdWithNoSubCmds(args);
+
+        private static bool NoArgsPassed(string[] args) => args.Length == 0;
+
+        private static bool NewCmdWithNoSubCmds(string[] args) => args.Length == 1 && args[0] == "new";
```

Comment:
This introduces coupling to the ``NewProjectCommand`` indirectly via it's command word. Could we update ``AutoStepCommand`` to allow a base 'command is valid' method perhaps?